src/util/helpers.js

```diff
@@ -136,3 +136,7 @@ export const getStartEndDateError = (startDate, endDate) => {
     }
     return null;
 };
+
+// Remove line breaks from text (not displayed corrently in map downloads)
+// https://stackoverflow.com/questions/10805125/how-to-remove-all-line-breaks-from-a-string/10805292#10805292
+export const removeLineBreaks = text => text.replace(/\r?\n|\r/g, '');
```

Comment:
Better, nice.  As that StackOverflow answer states though do we want to replace with a space instead of an empty string?  Is it possible there will be newlines between words?

Reply:
Should only be newlines between tags, but added a space in any case: